lxd/snapshot_common.go

```diff
@@ -6,21 +6,21 @@ import (
 	"strings"
 	"time"
 
-	"gopkg.in/robfig/cron.v2"
+	"github.com/robfig/cron/v3"
 
 	"github.com/lxc/lxd/lxd/util"
 )
 
 // SnapshotScheduleAliases contains the mapping of scheduling aliases to cron syntax
 // including placeholders for scheduled time obfuscation.
 var SnapshotScheduleAliases = map[string]string{
-	"@hourly":   "%s %s * * * *",
-	"@daily":    "* %s %s * * *",
-	"@midnight": "%s %s 0 * * *",
-	"@weekly":   "* %s %s * * 0",
-	"@monthly":  "* %s %s 1 * *",
-	"@annually": "* %s %s 1 1 *",
-	"@yearly":   "* %s %s 1 1 *",
+	"@hourly":   "%s * * * *",
```

Comment:
@stgraber has the cron format changed for end users now?

Reply:
No

Reply:
Cool, what has changed here?

Reply:
We never exposed second granularity to LXD users because it's out of spec for cron entries.
Upstream has now followed and similarly doesn't do second granularity.

So this change switches to the new v3 package and drops any place where we prepended a `* ` to the user provided strings as well as this spot where our built-in aliases were using second granularity for their random times.